app/grandchallenge/reader_studies/views.py

```diff
@@ -866,7 +866,11 @@ def get_context_data(self, **kwargs):
 class ReaderStudyViewSet(ReadOnlyModelViewSet):
     serializer_class = ReaderStudySerializer
     queryset = ReaderStudy.objects.all().prefetch_related(
-        "images", "questions__options"
+        "images",
+        "questions__options",
+        "display_sets",
+        "display_sets__values",
+        "display_sets__values__image",
```

Comment:
You can just do `"display_sets__values__images"`, all intermediates are returned too.

Reply:
Never knew that, thank you!